config/sync/views.view.gp_list.yml

```diff
@@ -0,0 +1,640 @@
+            operator_list: {  }
+            identifier: last_name
+            required: false
+            remember: false
+            multiple: false
+            remember_roles:
+              authenticated: authenticated
+              anonymous: '0'
+              administrator: '0'
+              gp_author_user: '0'
+              author_user: '0'
+              gp_supervisor_user: '0'
+              news_supervisor: '0'
+              supervisor_user: '0'
+              editor_user: '0'
+              admin_user: '0'
+              apps_user: '0'
+              health_condition_author_user: '0'
+              health_condition_supervisor_user: '0'
+              driving_instructor_supervisor_user: '0'
+            placeholder: ''
+          is_grouped: false
+          group_info:
+            label: ''
+            description: ''
+            identifier: ''
+            optional: true
+            widget: select
+            multiple: false
+            remember: false
+            default_group: All
+            default_group_multiple: {  }
+            group_items: {  }
+          entity_type: gp
+          entity_field: last_name
+          plugin_id: string
+      sorts:
+        last_name:
+          id: last_name
+          table: gp_field_data
+          field: last_name
+          relationship: none
+          group_type: group
+          admin_label: ''
+          order: ASC
+          exposed: false
+          expose:
+            label: ''
+          entity_type: gp
+          entity_field: last_name
+          plugin_id: standard
+        first_name:
+          id: first_name
+          table: gp_field_revision
+          field: first_name
+          relationship: none
+          group_type: group
+          admin_label: ''
+          order: ASC
+          exposed: false
+          expose:
+            label: ''
+          entity_type: gp
+          entity_field: first_name
+          plugin_id: standard
+      title: 'GP List'
+      header:
+        area:
+          id: area
+          table: views
+          field: area
+          relationship: none
+          group_type: group
+          admin_label: ''
+          empty: true
+          tokenize: false
+          content:
+            value: |
+              This page shows all the GPs listed on Nidirect.
+              <br><br>
+              This page was created by the web development team to assist with QA and verification of the GP list. It is not intended for other uses, or as a replacement for the master spreadsheet available from other sources.
+            format: basic_html
+          plugin_id: text
+      footer: {  }
+      empty: {  }
+      relationships: {  }
+      arguments: {  }
+      display_extenders: {  }
+    cache_metadata:
+      max-age: -1
+      contexts:
+        - 'languages:language_content'
+        - 'languages:language_interface'
+        - url
+        - url.query_args
+      tags: {  }
+  page_1:
```

Comment:
Probably worth renaming the default 'page_1' display ID to something more descriptive